Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2701 -- V4 frontend cleanup

Resolves [EREGCSC-2701](https://jiraent.cms.gov/browse/EREGCSC-2701)

**Description**

Make necessary changes to the front end and associated end to end tests so that the site works as expected.

**This pull request changes:**

- Provide proper parameters to API endpoints
- Handle updated API response
- Updates Cypress tests and fixtures

** This pull request does NOT change:**

- Vitest unit tests (will be addresses in follow-up ticket)
- JSDoc comments and type definitions (will be addresses in follow-up ticket)

**Steps to manually verify this change:**

1. Cypress tests pass in `deploy-experimental` GitHub Action
2. Visit [experimental deployment](https://typvl84r9b.execute-api.us-east-1.amazonaws.com/dev1351/)
3. Validate that site works as expected without any regressions
